flexget/utils/tools.py

```diff
@@ -426,3 +426,24 @@ def get_latest_flexget_version_number():
 
 def get_current_flexget_version():
     return flexget.__version__
+
+def data_size_to_bytes(size):
+    """
+    Expands a data size into its bytes as an integer i.e. '1 MB' becomes 1048576
```

Comment:
This is not entirely true assuming sane abbreviations.

1 MB is 1000000 bytes. 1 MiB is 1048576. I agree with @gazpachoking that the function should take an extra param that sets it to either base 2 or base 10.

If you need help getting these commits into its own branch/PR you know where to find us :)